django/utils/autoreload.py

```diff
@@ -151,6 +153,9 @@ def iter_modules_and_files(modules, extra_files):
             )
             sys_file_paths.append(origin)
 
+    stdlib_location_1 = os.path.dirname(os.__file__)
```

Comment:
Not all implementations of Python have even the standard library in files. `os.__file__` may not exist.